src/context/AppProvider/AppContext.tsx

```diff
@@ -195,13 +196,15 @@ export const AppProvider = ({ children }: { children: React.ReactNode }) => {
     })
   }, [dispatch, accountSpecifiersList])
 
-  // once portfolio is done loading, fetch all transaction history
+  // once portfolio is done loading, fetch all transaction history and apr data
```

Comment:
We probably want to have another `useEffect` for this, there's no need to wait for portfolio load to start loading APR data